src/SatisPress.php

```diff
@@ -210,7 +210,11 @@ public function query_vars( $vars ) {
 	 * @param Package $package Package object.
 	 * @param string  $version Optional. Version of the package to send. Defaults to the current version.
 	 */
-	protected function send_package( $package, $version = '' ) {
+	protected function send_package( $package, $version = null ) {
+		if ( null === $version ) {
+			$version = '';
+		}
```

Comment:
What's the rationale for switching all these defaults to `null` and adding a conditional like this?

Reply:
Admittedly opinionated, but as per the commit message: https://github.com/kalessil/phpinspectionsea/blob/master/docs/code-style.md#non-null-parameters-default-value

Keeps the logic of what the default value is, inside the function/method, and isn't part of the signature. Allows for more flexibility should that default value ever change. Allows use of `?string` type declaration for PHP 7.1+.

Now it's PHP 7.0+, this conditional could become `$version = $version ?? '';` and when https://wiki.php.net/rfc/null_coalesce_equal_operator gets added, it would be `$version ??= '';`

The format, as it is right now, is a stepping stone to what it could be, once min PHP version is high enough :-)